### PR TITLE
Changes to the Volunteer Application Form

### PR DESCRIPTION
**Pull request by jeyanthi-sm**
Zenhub - https://app.zenhub.com/workspaces/devrecruit-team-644b85772003020cab70d47e/issues/gh/dev-launchers/dev-launchers-platform/1898

To match the Figma Styling the submit button and to incorporate the functionality of submit button
Added the 'isAgeOver18' and 'isTermsAgreed' columns to the Applicant Schema and the corresponding front-end changes

### CODE REVIEW

**Comment by jeyanthi-sm** [comment]
While using the checkbox in my page the tick in the checkbox and the background color are white so I updated the color from white to black. In the file -  packages/UI/src/components/atoms/Checkbox/StyledCheckbox.ts doing so made the tick visible and i was able to complete the page. However the build is failing. Can anybody help point why/how to proceed to resolve this?

**Review by ajaywayase — APPROVED** [review]

File: packages/UI/tsconfig.json
Comment (Enjoy2Live) [inline packages/UI/tsconfig.json:13]: @jeyanthi-sm @ajaywayase always make sure that code owners review your changes before merging.

Please delete the outDir.

**Comment by Enjoy2Live** [comment]
> While using the checkbox in my page the tick in the checkbox and the background color are white so I updated the color from white to black. In the file - packages/UI/src/components/atoms/Checkbox/StyledCheckbox.ts doing so made the tick visible and i was able to complete the page. However the build is failing. Can anybody help point why/how to proceed to resolve this?

Can I see screenshot of the how it used to look like before your changes? There's a chance your change will break on other machines because most devices have the tick emoji as black but for you it was white. Please don't merge until you've received an approval from code owners.

**Comment by jeyanthi-sm** [comment]
Please find attached the screenshot - Screenshot 2024-08-23 112917

**Comment by jeyanthi-sm** [comment]
Checkbox After change screenshot  
Screenshot 2024-08-23 113354

**Pull request merged**
